programs/mpl-core/src/plugins/lifecycle.rs

```diff
@@ -46,7 +47,7 @@ impl ExternalCheckResult {
     }
 }
 
-/// Bitfield representation of lifecycle permissions for external, third party plugins.
+/// Bitfield representation of lifecycle permissions for adapter, third party plugins.
```

Comment:
This comment is a bit strange

Reply:
Additionally, maybe you should do a search for "third party" and remove those references?